karaf/features/src/main/feature/feature.xml

```diff
@@ -208,6 +208,7 @@
         <feature>cxf-jaxrs</feature>
         <bundle dependency="true">mvn:org.apache.cxf/cxf-rt-rs-security-cors/${cxf.version}</bundle>
         <bundle dependency="true">mvn:org.apache.cxf/cxf-rt-frontend-jaxrs/${cxf.version}</bundle>
+        <bundle dependency="true">mvn:org.apache.cxf.karaf/cxf-karaf-commands/3.2.7</bundle>
```

Comment:
Could you change this to `${cxf.version}`?

Reply:
Good point. Actually, I think you can reuse `${cxf.version}` directly as these bundles are released in the same time

Reply:
@tbouron I see now that the version of cxf is a typo in the PR description. Use `${cxf.version}`

Reply:
oops - sorry about that